internal/osbuildconfig/osbuildconfig_handler.go

```diff
@@ -0,0 +1,95 @@
+// using --old-style-config flag in order to separate the generated files for each type (types, server, and client files)
+//go:generate go run -mod=mod github.com/deepmap/oapi-codegen/cmd/oapi-codegen -package=restapi -old-config-style -generate=types -o ../../restapi/osbuildconfig_webhook_trigger_types.go  ../../osbuildconfig_webhook_trigger.yaml
+//go:generate go run -mod=mod github.com/deepmap/oapi-codegen/cmd/oapi-codegen -package=restapi -old-config-style -generate=chi-server -o ../../restapi/osbuildconfig_webhook_trigger_server.go  ../../osbuildconfig_webhook_trigger.yaml
+//go:generate go run -mod=mod github.com/deepmap/oapi-codegen/cmd/oapi-codegen -package=restapi -old-config-style -generate=client -o ../../restapi/osbuildconfig_webhook_trigger_client.go  ../../osbuildconfig_webhook_trigger.yaml
+
+package osbuildconfig
+
+import (
+	"fmt"
+	"net/http"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/project-flotta/osbuild-operator/internal/httpapi"
+	loggerutil "github.com/project-flotta/osbuild-operator/internal/logger"
+	"github.com/project-flotta/osbuild-operator/internal/manifests"
+	repositoryosbuildconfig "github.com/project-flotta/osbuild-operator/internal/repository/osbuildconfig"
+	"github.com/project-flotta/osbuild-operator/internal/repository/secret"
+	"github.com/project-flotta/osbuild-operator/restapi"
+)
+
+type OSBuildConfigHandler struct {
+	OSBuildConfigRepository repositoryosbuildconfig.Repository
+	SecretRepository        secret.Repository
+	OSBuildCRCreator        manifests.OSBuildCRCreator
+}
+
+func NewOSBuildConfigHandler(osBuildConfigRepository repositoryosbuildconfig.Repository,
+	secretRepository secret.Repository, osBuildCRCreator manifests.OSBuildCRCreator) *OSBuildConfigHandler {
+	err := httpapi.Load()
```

Comment:
Why do you need to load the configuration here? it was already loaded in main

Reply:
removed it